operator/src/main/java/oracle/kubernetes/operator/helpers/DomainPresenceInfo.java

```diff
@@ -54,6 +55,7 @@
   private final ConcurrentMap<String, V1Service> clusters = new ConcurrentHashMap<>();
 
   private final List<String> validationWarnings = Collections.synchronizedList(new ArrayList<>());
+  private EventItem eventType;
```

Comment:
This records the last EventType, so it would be clearer if we name the variable lastEventType.

Reply:
ok, I'll change the variable and method names. I'm thinking to change the variable name to `lastEventItem` since it's of type `EventItem`. Any thoughts? Thanks.

Reply:
that is fine.